code/modules/clothing/masks/_masks.dm

```diff
@@ -26,6 +26,17 @@
 	. = ..()
 	UnregisterSignal(M, COMSIG_MOB_SAY)
 
+/obj/item/clothing/mask/vv_edit_var(vname, vval)
+	if(vname == NAMEOF(src, modifies_speech) && ismob(loc))
+		var/mob/M = loc
+		if(M.get_item_by_slot(ITEM_SLOT_MASK) == src)
+			if(vval)
+				if(!modifies_speech)
+					RegisterSignal(M, COMSIG_MOB_SAY, .proc/handle_speech)
+			else if(modifies_speech)
+				UnregisterSignal(M, COMSIG_MOB_SAY)
+	return ..()
+
 /obj/item/clothing/mask/proc/handle_speech()
```

Comment:
If this is being used as a signal handler now, it needs to have `SIGNAL_HANDLER` on it.

Reply:
Actually, it appears that this being a signal handler isn't new. Can you add it anyway if it doesn't break anything?

Reply:
a ton of handle_speech and handle_hearing procs are missing it, i'll leave that for another PR though